Windows/MainWindow.xaml.cs

```diff
@@ -149,7 +173,7 @@ public partial class MainWindow {
 
             UpdateCheck.Run(this);
 
-            TryLoad(args);
+            // TryLoad(args);
```

Comment:
Check for some batch export cmd flag instead, that way you don't disable this feature.

Reply:
I wasn't sure how this worked, sorry.